src/pages/index.js

```diff
@@ -50,12 +55,15 @@ export async function getServerSideProps(context) {
     for (const memberData of members) {
       const { picture, username, isMember, first_name, archivedMember } =
         memberData;
-      const img_url = picture
-        ? getCloudinaryImgURL(
-            picture.publicId,
-            `${WIDTH_200PX},${HEIGHT_200PX}`
-          )
-        : getImgURL(username, 'img.png');
+
+      let img_url =
+        dev && picture
+          ? getCloudinaryImgURL(
+              picture.publicId,
+              `${WIDTH_200PX},${HEIGHT_200PX}`
+            )
+          : '';
+      if (!dev) img_url = getImgURL(username, 'img.png');
```

Comment:
**Suggestion:** When dev is true, for users without Cloudinary images we should add a source of the default Avatar image instead of an empty string

Reply:
Why is the logic of displaying member images different from the individual member page? 
Reference: [src/pages/[id]/index.js](https://github.com/Real-Dev-Squad/website-members/blob/5e26323e25110b1b13f6154c2c9f352d369c4edb/src/pages/%5Bid%5D/index.js#L76-L80)

Reply:
For the second question as discussed: We are using the feature flag `dev=true` to let members know if their profile picture is properly set or not, so we don't want to fetch images from `website-static` repo and avoid confusion.

Reply:
@shubham-y default image added